_includes/tutorials/aggregating-sum/ksql/markup/dev/create-movie-ticket-sales.adoc

```diff
@@ -0,0 +1,7 @@
+First, you'll need to create a Kafka topic and stream to represent the ticket sales. 
+The statement below creates both at the same time. This stream contains the name of the movie and the price of the ticket to watch it. We model the price as an integer to make the example simple.
+Another important characteristic of the data is the timestamp column, `sale_ts`, which can be used for time-based windowing. KSQL can use either the Kafka message timestamp or a field from the message value as the timestamp. In this example, we'll use the latter, which is also known as _event time_.
```

Comment:
One other thing I noticed is that this tutorial goes out of its way to explain event time, but doesn't use it. Did you intend to use it in the queries? Worth leaving out if not.

Reply:
Good point. There is no window, so no real use for event time. Might as well just nuke it and the explanation, I think.